python/adelphi/adelphi/export.py

```diff
@@ -0,0 +1,100 @@
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import hashlib
+import logging
+from base64 import urlsafe_b64encode
+from datetime import datetime, tzinfo, timedelta
+
+try:
+    from itertools import ifilterfalse as filterfalse
+except ImportError:
+    from itertools import filterfalse
+
+from adelphi import NoKeyspacesSelectedException
+from adelphi.anonymize import anonymize_keyspace
+from adelphi.store import build_keyspace_objects
+
+
+log = logging.getLogger('adelphi')
+
+try:
+    from datetime import timezone
+    utc = timezone.utc
+except ImportError:
+    class UTC(tzinfo):
+        def utcoffset(self, dt):
+            return timedelta(0)
+        def tzname(self, dt):
+            return "UTC"
+        def dst(self, dt):
+            return timedelta(0)
+    utc = UTC()
+
+class BaseExporter:
+    
+    # unique_everseen from itertools recipes
+    def __unique(self, iterable, key=None):
```

Comment:
There was no plausible reason to make this visible beyond the use here in get_cluster_metadata